Python/ceval_gil.c

```diff
@@ -1319,5 +1319,70 @@ _Py_HandlePending(PyThreadState *tstate)
             return -1;
         }
     }
+
+#ifdef Py_REMOTE_DEBUG
+    const PyConfig *config = _PyInterpreterState_GetConfig(tstate->interp);
+    if (config->remote_debug) {
```

Comment:
Might be worth doing:
```suggestion
    if (config->remote_debug == 1) {
```
to harden this at least a little bit against heap corruption.